Introductory Tests/UDP_basic.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    prob = pg.problem(basic_udp())
</pre><p>before you go into optimizing the solution I would run a few test calls of the fitness function as a sanity check to see if the values make sense</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>